platforms/documentation/docs/src/docs/userguide/optimizing-performance/file_system_watching.adoc

```diff
@@ -48,11 +48,11 @@ Gradle supports file system watching on the following operating systems:
 
 * Windows 10, version 1709 and later
 * Linux, tested on the following distributions:
-** Ubuntu 16.04 or later
-** CentOS Stream 8 or later
-** Red Hat Enterprise Linux (RHEL) 8 or later
-** Amazon Linux 2 or later
-** Alpine Linux 3.20 or later
+** Ubuntu 16.04
+** CentOS Stream 9
+** Red Hat Enterprise Linux (RHEL) 8
+** Amazon Linux 2
+** Alpine Linux 3.20
```

Comment:
💅 The way we set up Alpine tests in [gradle-fileevents](https://github.com/gradle/gradle-fileevents/blob/3896c08914b7273d50e04c472c23782431f28921/.teamcity/src/main/kotlin/Platform.kt#L20) makes the Alpine version depend (and change with) the eclipse-temurin container, it may use a later version at any moment.

The Alpine tests running in `gradle/gradle` take the Alpine version from the TD agent image that will also be updated without anyone keeping this comment in sync.

I don't expect us to run tests on old Alpine versions, would it be better to remove the version here?

Reply:
Let's figure out how we want to update these versions separately. It's not Alpine only that depends on external changes (e.g. CentOS Stream 8 has also been dropped without any trigger to update this).

FTR, here we should only claim support for OSs we run tests on for `gradle/gradle`.